tool/matomo/README.md

```diff
@@ -0,0 +1,22 @@
+# Matomo (formerly Piwik)
+- Set the Site ID
+- Choose whether you want image fallback tracking
+- Enter the URL to your Matomo install excluding http/https and trailing slashes
+- Choose whether you want to track admins (not recommended)
```

Comment:
A bit off topic but can anyone point me at a reason for this? I would always want to record all traffic, seems pretty arbitrary to exclude admins. I’d love to see this removed but failing that I think it should default so all traffic is recorded

Reply:
(Can be later issue)

Reply:
I think that was a copy/paste functionality from the existing https://github.com/bmbrands/moodle-local_analytics
Agree. Should be removed.